libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalAggregateView/CausalAggregateStyles.ts

```diff
@@ -42,6 +42,7 @@ export const CausalAggregateStyles: () => IProcessedStyleSet<ICausalAggregateSty
         display: "inline-block",
         flex: "1",
         fontSize: 14,
+        maxWidth: "750px",
```

Comment:
Can we define the string "750px" some where in one common place. It might be easier to understand that the info text needs to be of the same size for all components.

Reply:
Great point! Let me define it in a common place so each component can share.

Reply:
Updated